src/library/dao/playlistdao.cpp

```diff
@@ -246,6 +246,29 @@ bool PlaylistDAO::isPlaylistLocked(const int playlistId) const {
     return false;
 }
 
+bool PlaylistDAO::clearPlaylist(const int playlistId) {
+    int position =
+        (m_pAutoDJProcessor && m_pAutoDJProcessor->nextTrackLoaded()) ? 2 : 1;
```

Comment:
When replacing the autoDJ queue, the first track is retained if it is loaded in a deck. Is it the desired behavior?